app.lua

```diff
@@ -68,7 +68,10 @@ package.loaded.capture_errors = function(fn)
 end
 
 require 'models'
+require 'models.users'
```

Comment:
Eventually, I think all models should be their own files and we can autoload `models/`